spec/lib/myfinance/resources/entity_spec.rb

```diff
@@ -52,8 +52,8 @@
     end
 
     context "when not found" do
-      before { Myfinance.configuration.token = "wrong-token" }
-      subject { described_class.find(3798) }
+      let(:client) { Myfinance.client('') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.